src/Datadog.Trace.MSBuild/Datadog.Trace.MSBuild.csproj

```
@@ -6,15 +6,15 @@
 
   <!-- For VS testing purposes only, copy all implementation assemblies to the
        to test the loading of managed profiler assemblies from disk -->
-  <PropertyGroup Condition="'$(BuildingInsideVisualStudio)'=='true' and '$(TargetFramework)' == 'netstandard2.0'">
+  <PropertyGroup Condition="'$(BuildingInsideVisualStudio)'=='true' and ('$(TargetFramework)' == 'netstandard2.0' OR '$(TargetFramework)' == 'netcoreapp3.1')">
```

nit (you even fixed it in `Datadog.Trace.ClrProfiler.Managed.csproj` 😉)
```suggestion
  <PropertyGroup Condition="'$(BuildingInsideVisualStudio)'=='true' AND ('$(TargetFramework)' == 'netstandard2.0' OR '$(TargetFramework)' == 'netcoreapp3.1')">
```